Gems/ImGui/Code/Source/LYCommonMenu/ImGuiLYCommonMenu.cpp

```diff
@@ -294,7 +311,8 @@ namespace ImGui
                         ImGui::TextColored(ImGui::Colors::s_PlainLabelColor, "Load Level: ");
                         for (int i = 0; i < levelNames.size(); i++)
                         {
-                            if (ImGui::MenuItem(AZStd::string::format("%d- %s", i, levelNames[i].c_str()).c_str()))
+                            bool isNetworked = networkedLevelNames.contains(levelNames[i]);
+                            if (ImGui::MenuItem(AZStd::string::format("%d- %s%s", i, levelNames[i].c_str(), isNetworked ? " (Networked)":"").c_str()))
```

Comment:
IMHO "Networked" seems like an odd term. I want to ask "Networked in what way?" We are really trying to indicate this thing needs a server/host (so expect there to be some kind of process that is acting as server). But I can't think of a better term thats just as simple.

Reply:
Hmm, good point. How about "Multiplayer". I chose networked just because those spawnables have network.spawnables siblings. If no suffix description is better, I could just leave it .spawnable (as it was before). 
This PR was mainly to remove the .network.spawnable because no one should be loading those ever, so I'd be happy with just having that :P See which levels are networked or not was just icing on the cake.

Reply:
I agree that the term "multiplayer" feel more in alignment with the terminology used in the GEM itself as well as on the documentation. "Networking" feels too general and doesn't say not enough about what it does.

Reply:
For future context, we should also not specify what it does, but more about what it is. I'm a "multiplayer" related file. And stay away from, I deal with... Example (Asynchronous resolution). Too much detail for a file categorization. Feel free to disagree